scripts/release/package.sh

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env bash
+# shellcheck disable=2164,2166
+
```

Comment:
I think that you're missing `set -e` here.

Reply:
Will set.